### DISCUSSION TITLE
Welcome to Stargate Journey Discussions!

### DISCUSSION THREAD

**Discussion by Povstalec** [Povstalec/StargateJourney#23] [Announcements]
## 👋 Welcome!
So yeah, I seriously didn't realize the discussions were turned off. But hey, better late than never. You can go ahead and ask/talk about whatever you want, there are no limitations really.

**Comment by Dezzie9889** [comment]
the pic for how to make the gate is missing from the wiki and I cant figure out how to make it.

**Reply by Povstalec** [reply]
Not sure, why it disappeared for no reason, but I fixed it: https://github.com/Povstalec/StargateJourney/wiki/Getting-Started#creating-your-own-stargate
 Thanks for notifying me.

**Reply by Dezzie9889** [reply]
still not showing for me, just the dates of the pic uploads.

**Reply by Povstalec** [reply]
Well that's annoying, I don't know why github is acting up.

Either way, you should be able to find it here as one of the last images: https://legacy.curseforge.com/minecraft/mc-mods/sgjourney/screenshots
